### PR TITLE
Make ES6 opt-out instead of opt-in

### PR DESCRIPTION
**Pull request by SimenB**
/cc @sverosak @peternic @CMTegner

Last one before I think we should try applying it to different projects.

### CODE REVIEW

File: packages/eslint-config-finn/es6.js [hunk 1]
```diff
@@ -1,6 +0,0 @@
-'use strict';
```
Comment (SimenB) [inline packages/eslint-config-finn/es6.js]: Super breaking probably, but spt already did this, so `eslint-config-spt/env-es6` was already included...

**Comment by CMTegner** [comment]
LGTM

**Comment by sveisvei** [comment]
Breaking is I guess ok as long as we do make it easy to fix :).

wfm!

**Comment by theneva** [comment]
lgtm.

**Pull request merged**